### DISCUSSION TITLE
When Maxwell synchronizes binlogs, does it still require authorization to filter out databases that do not need to be synchronized?

### DISCUSSION THREAD

**Discussion by jackfeng1986** [zendesk/maxwell#2059] [Q&A]
1) Without authorization, when Maxwell starts, an error is reported: Couldn't find database xxx_ DbName.

2) If authorization is required for all, is there a risk of data access?

**Comment by osheroff** [comment]
1.  you can use the `blacklist` feature of filters to avoid databases you don't need

2. The standard SQL connection that would need permission to view the database isn't used for very much.  If someone managed to trigger a "bootstrap" they could access data from the other database but otherwise I'd think it's fairly safe..
